publish.sh

```diff
@@ -8,3 +10,11 @@ docker pull registry.tld/conjurinc/publish-rubygem
 summon --yaml "RUBYGEMS_API_KEY: !var rubygems/api-key" \
   docker run --rm --env-file @SUMMONENVFILE -v "$(pwd)":/opt/src \
   registry.tld/conjurinc/publish-rubygem parse_a_changelog
+
+bl_info "Published ${repo_gem_version} to rubygems.org"
```

Comment:
where does repo_gem_version get set?

Reply:
Oops that line should have been removed, `bl_info` isn't available either.

Reply:
👍 ok, let me know when it's ready for re-review

Reply:
Thanks ready again @izgeri